src/js/serial_backend.js

```diff
@@ -91,7 +91,7 @@ function initializeSerialBackend() {
                         finishClose(toggleStatus);
                     }
 
-                    mspHelper.setArmingEnabled(true, false, onFinishCallback);
+                    mspHelper?.setArmingEnabled(true, false, onFinishCallback);
```

Comment:
Why is `setArmingEnabled `not available here?

Reply:
i have no idea, but if i dont do that, the app wont start port_handler

Reply:
@chmelevskij any idea?

Reply:
https://github.com/betaflight/betaflight-configurator/blob/54738ef833d2a5819caca20ee3a5fc613cc7c617/src/js/serial_backend.js#L3

It's not defined at first, so might be something is async and race condition breaks it.

Reply:
```suggestion
                    mspHelper.setArmingEnabled(true, false, onFinishCallback);
```
This no longer is needed.